### ISSUE TITLE
I want to use my own model

### ISSUE THREAD

**Issue by YujeongShin570** [JaidedAI/EasyOCR#186]
Hi, there!
I want to use my own model in EasyOCR.
My own model's file format is .pth. Can I just put my .pth file in "easyocr/model" folder? Will it work find?

Thank you for your help 👍

**Comment by ghandic** [comment]
What model have you trained? Could you give more details? If it is a custom trained CRAFT model it should work.

**Comment by YujeongShin570** [comment]
@ghandic 
I read one issue in here that easyocr made a model with using https://github.com/clovaai/deep-text-recognition-benchmark. So, I follow that one!

**Comment by ghandic** [comment]
Yes, that's the recognition model, I haven't tested loading a new model but you should be able to do so. Give it a go and let us know if it works/trace back of the error if it doesn't

**Comment by YujeongShin570** [comment]
@ghandic  Thank you for quick response : ) After I tested my model, I will let you know.

**Comment by rkcosmos** [comment]
In order to use custom model, you need to make sure your weight has exactly the same structure as what we are using. Also I don't remember if I made other changes in training script.

**Comment by YujeongShin570** [comment]
@rkcosmos Hi, I made custom model and it is made as TPS, ResNet, LSTM and CTC. When I moved best_accuracy.pth to ./EasyOCR/model and renamed it to korean.pth. Whenever I run the code, korean.pth is changed to your korean model. Do I have to edit some codes? Would you give me some advice? I am very struggling with it : (

**Comment by rkcosmos** [comment]
1. TPS is not going to work because it is only for fix aspect ratio image (which is not the case for general OCR)
2. yes, you have to edit some codes. Probably a lot, depending on the model you have and parameters you set. I didn't use default values when I trained model for EasyOCR. If you really want to use your model now, you have to read and understand my code.
